namespaces/main.go

```diff
@@ -180,14 +181,24 @@ func getSession() (*session.Session, error) {
 }
 
 func getAwsNamespaces() ([]string, error) {
-	nsStr := os.Getenv(envAwsNamespaces)
-	if nsStr == emptyString {
-		return nil, fmt.Errorf("env %s must be set", envAwsNamespaces)
+	awsNsStr := os.Getenv(envAwsNamespaces)
+	customNs := os.Getenv(envCustomNamespace)
+
+	if awsNsStr == emptyString && customNs == emptyString {
+		return nil, fmt.Errorf("either %s or %s must be set", envAwsNamespaces, envCustomNamespace)
+	}
+
+	fullNsStr := awsNsStr
+	if customNs != emptyString {
+		if fullNsStr != emptyString {
```

Comment:
I'm not a fan of how this condition looks... it's not straight forward in my opinion
Please change it to be more readable :)